src/sample/Main.java

```diff
@@ -32,29 +34,28 @@ static void setupView(View gameView)
 
         // setting up the buttons which go into the UI
         gameView.setupButtons(
-                (o) -> game.executeAutoMove(),
+                (o) -> game.MakeMove(),
```

Comment:
We do still need to work to get rid of this pattern soon. I think it is a task. I will still approve but it just reminded me.